apps/web/components/ui/VsfLogo.vue

```diff
@@ -1,10 +1,33 @@
 <template>
   <picture>
-    <NuxtImg :src="imagePath" alt="plentyShop PWA Logo" class="w-100 h-10 py-2" width="150" height="40" preload />
+    <template v-if="imageExtension === 'svg'">
+      <NuxtImg :src="imagePath" alt="Logo" class="w-100 h-10 py-2" width="150" height="40" preload />
```

Comment:
Just to clarify, if you use an SVG image, you are bound by the 150x40 dimensions, right?

Reply:
Answer is yes.